Makefile

```diff
@@ -29,7 +29,6 @@ _touch:
 	rm -fr $(ROOT)/edgedb/protocol/protocol.c
 	find $(ROOT)/edgedb/protocol -name '*.pyx' | xargs touch
 	find $(ROOT)/edgedb/datatypes -name '*.pyx' | xargs touch
-	find $(ROOT)/edgedb/datatypes -name '*.c' | xargs touch
```

Comment:
Is this supposed to be part of this commit?

Reply:
Not really, but that line is questionable as it cancels the effect of the preceding .pyx touch.